src/main/java/hw6/pages/UserTablePage.java

```diff
@@ -0,0 +1,58 @@
+package hw6.pages;
+
+import hw6.pages.components.HeaderSectionComponent;
+import hw6.pages.components.LeftSectionMenuComponent;
+import hw6.pages.components.UsersTableComponent;
+import hw6.pages.components.UsersTableLogsComponent;
+
+import java.util.List;
+
+public class UserTablePage extends AbstractPage {
+
+    UsersTableComponent usersTableComponent;
+    UsersTableLogsComponent usersTableLogsComponent;
+
+    @Override
+    void initElementsOnPage() {
+        this.headerSectionComponent = new HeaderSectionComponent();
+        this.leftSectionMenuComponent = new LeftSectionMenuComponent();
```

Comment:
I suppose, that initialization these elements could be moved to the `AbstractPage`